tensorflow/python/keras/saving/hdf5_format.py

```diff
@@ -35,6 +35,7 @@
 from tensorflow.python.platform import tf_logging as logging
 from tensorflow.python.util import serialization
 from tensorflow.python.util.lazy_loader import LazyLoader
+import tensorflow.python.platform.gfile as gfile
```

Comment:
Use `from .... import ...`. 

Also, don't forget to re-order the imports alphabetically, so this line should go before `from tensorflow.python.platform import tf_logging as logging`